client/src/js/components/bhReceipt.js

```diff
@@ -1,34 +1,34 @@
-var bhReceiptTemplate =
-  '<a ng-click="$ctrl.open()" href="">{{ $ctrl.displayValue }}</a>';
+const bhReceiptTemplate = `
+  <a ng-click="$ctrl.open()" href="">
+    {{ $ctrl.displayValue }}
+  </a>`;
 
 angular.module('bhima.components')
   .component('bhReceipt', {
     template : bhReceiptTemplate,
     controller  : bhReceiptController,
     bindings    : {
       value : '<',
-      displayValue : '<',
       type : '@',
+      displayValue : '<',
```

Comment:
Why is it restored in this PR ? Because in this PR i don't see where it is used.

Reply:
It wasn't restored, just moved from the second position to the third.  It is used in the template on line 1.